docs/kubernetes.md

```diff
@@ -55,7 +55,7 @@ helm upgrade npm verdaccio/verdaccio
 ### Uninstalling
 
 ```bash
-helm del --purge npm
+helm uninstall npm
```

Comment:
About this one, which one is correct https://github.com/verdaccio/charts#install-verdaccio-chart ?

Reply:
So this is the old one. I basically correted the README to be compatible w helm 3.

Reply:
thanks for the clarification

Reply:
i get your comment now. 
I check that also!

Reply:
https://github.com/verdaccio/charts/pull/38